### PR TITLE
added stackrox to argocd

### PR DESCRIPTION
**Pull request by traytorous**

### CODE REVIEW

File: argocd/overlays/moc-infra/applications/envs/moc/smaug/operate-first/stackrox.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: stackrox
+spec:
+  project: operate-first
+  source:
+    repoURL: https://github.com/operate-first/apps.git
+    targetRevision: HEAD
+    path: stackrox/overlays/moc/smaug
```
Comment (HumairAK) [inline argocd/overlays/moc-infra/applications/envs/moc/smaug/operate-first/stackrox.yaml:10]: this path doesn't exist

**Comment by traytorous** [comment]
This commit will be added when I upload the stackrox manifest. It should be the last step.

**Comment by Gregory-Pereira** [comment]
/lgtm
This looks production ready, I would approve this, but this depends on the manifests pr otherwise the argocd path wont exist.
/hold on #2211

**Pull request closed without merging**